includes/class-wc-payments.php

```diff
@@ -1622,6 +1622,10 @@ function ( $container ) {
 	 * @return void
 	 */
 	public static function maybe_init_woopay_direct_checkout() {
+		if ( ! self::$woopay_util->should_enable_woopay_on_cart_or_checkout() ) {
+			return;
+		}
+
```

Comment:
The code makes sense to me but I still can not reproduce the issue. Can you check my recording and see if anything I missed? Thank you!

https://github.com/user-attachments/assets/fb6f2d52-1844-400c-bedb-2d4f85b4e6db